python/jmxquery/__init__.py

```diff
@@ -110,6 +111,59 @@ def to_string(self):
 
         return string
 
+class JMXParam:
+    """
+    A JMX Parameter for JMXMethod.
+    """
+
+    def __init__(self,
+                 value: str="",
+                 value_type: str = "String"):
+        """
+        This method creates a JMXMethod param
+        :param value:
+        :param value_type:
+        """
+        self.value = value
+        self.value_type = value_type
+
+class JMXMethod:
+    """
+    A JMX Method which is used to invoke a specific MBean method from the JVM.
+    """
+
+    def __init__(self,
+                 m_bean_name: str,
+                 method_name: str = None,
+                 params: List[JMXParam] = []):
+        """
+         Creates instance of JMXMethod
+
+         :param m_bean_name:  The JMX MBean name. E.g.
+         :param method_name:  JMX method name
+         :param params:    a list of JMXParam parameters
+         """
+        self.mBeanName = m_bean_name
+        self.methodName = method_name
+        self.params = params
+
+    def json(self):
+        """
+        this method return the json serialization of this object
+        :return: json string
+        """
+        json_param = ",".join(["{\"value\":\""+param.value+"\", type:\""+param.value_type+"\"}" for param in self.params])
```

Comment:
This JSON is incorrect that occurred an error in Java.
```suggestion
        json_param = ",".join(["{\"value\":\""+param.value+"\"," type":\""+param.value_type+"\"}" for param in self.params])
```